api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md

```diff
@@ -35,7 +35,11 @@ This function allows you to customize the toolbar. Depending on the configuratio
 - [*"groupPanel"*](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/groupPanel '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/groupPanel/') 
 - [*"searchPanel"*](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/searchPanel '{basewidgetpath}/Configuration/searchPanel/')
 
-The following code shows how you can customize the toolbar using this function.
+All these items are [Buttons](/Documentation/ApiReference/UI_Widgets/dxButton/) and you can adjust their settings in the **options** object. The exception is the *"exportButton"* item. It does not have the *"options"* parameter because it is based on the [template](/Documentation/ApiReference/UI_Widgets/dxToolbar/Configuration/items/#template) funсtion. If you need to adjust export button options, implement a custom button instead.
```

Comment:
There's a little problem: `groupPanel` and `searchPanel` are not buttons. There's another problem: we tell how to customize the `exportButton`, but do not show it. We should either show or do not mention this at all. That's why I suggested (and still do) to fix only minor things in the code sample.